content/metadata.yaml

```
@@ -130,4 +130,12 @@ authors:
     affiliations:
       - Center for Epigenetics, Van Andel Research Institute
       - Department of Translational Genomics, Keck School of Medicine, University of Southern California
+   -
```

```suggestion
  -
```
This may be enough to fix the build error